src/DI.php

```diff
@@ -71,13 +71,20 @@ public static function getDice()
 	//
 
 	/**
+	 * @deprecated 2024.12 use DI::apphelper() instead
```

Comment:
Style preference, I feel like it reads better.
```suggestion
	 * @deprecated since 2024.12 use DI::apphelper() instead
```

Reply:
I feel the same, but thats not how the phpdoc syntax works:

> Syntax
> 
> `@deprecated [<Semantic Version>] [<description>]`

https://docs.phpdoc.org/guide/references/phpdoc/tags/deprecated.html